### PR TITLE
Material ui voyage details container

### PR DESCRIPTION
**Pull request by Swapna-Sahu**
Added Material-UI design for below files:
VesselDetails
CostWidgets
SuggestedRouteTable

### CODE REVIEW

**Review by JustinaOyov — COMMENTED** [review]
Nicely wired UI. Loads fine in both Storybook and React app.

**Review by JustinaOyov — DISMISSED** [review]
Nicely wired UI. Loads fine in both Storybook and React app.

File: src/client/components/VoyageDetails.js [hunk 1]
```diff
@@ -16,30 +51,60 @@ function VoyageDetails({
   hsfo,
   ulsfo
 }) {
+  const classes = useStyles();
   return (
-    <div>
+    <Card className={classes.card}>
```
Comment (zkwsk) [inline src/client/components/VoyageDetails.js]: Please do not put a card around the entire component. The teaser already has its own card. The result is that we get multiple cards nested inside each other.

Instead wrap the section below it and the selected routes table.

Like this: 

Screen Shot 2019-10-21 at 11 20 52

**Review by zkwsk — CHANGES_REQUESTED** [review]

File: src/client/components/VoyageDetails.js (hunk 1 above)
Comment (Swapna-Sahu) [reply src/client/components/VoyageDetails.js]: Resolved.

**Review by zkwsk — APPROVED** [review]

**Pull request merged**